science/iqtree/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+name                    iqtree
+github.setup            Cibiv IQ-TREE 1.6.12 v
+revision                0
+checksums               rmd160  249414b567bc4f3d2a939d8f7e71cd72f653ed3c \
+                        sha256  9614092de7a157de82c9cc402b19cc8bfa0cb0ffc93b91817875c2b4bb46a284 \
+                        size    4864641
+
+categories              science
+platforms               darwin
+maintainers             nomaintainer
+license                 GPL-2
+
+description             Efficient phylogenomic software by maximum likelihood
+
+long_description        The IQ-TREE software was created as the successor of IQPNNI and \
+                        TREE-PUZZLE (thus the name IQ-TREE). IQ-TREE was motivated by the \
+                        rapid accumulation of phylogenomic data, leading to a need for \
+                        efficient phylogenomic software that can handle a large amount of \
+                        data and provide more complex models of sequence evolution. To this \
+                        end, IQ-TREE can utilize multicore computers and distributed parallel \
+                        computing to speed up the analysis. IQ-TREE automatically performs \
+                        checkpointing to resume an interrupted analysis.
+
+homepage                http://www.iqtree.org/
+github.master_sites     ${github.homepage}/archive/
+
+depends_lib-append      port:eigen3 \
+                        port:libomp \
+                        port:zlib
+
+compiler.cpath-append   ${prefix}/include/libomp
+
```

Comment:
the build system also adds `-std=c++11`, so you likely want to require a compiler that supports this; that can be done by adding  `compiler.cxx_standard   2011`